detection-rules/attachment_adobe_image_lure_fts.yml

```diff
@@ -0,0 +1,32 @@
+name: "Attachment: Adobe image lure with with suspicious link from first time sender"
```

Comment:
do we want to add a check for a suspicious link? right now we're just looking for phrases and existence of a link

Reply:
we don't check other rules with similar intent.. eg DocuSign. Testing validates it's uneeded.